### PR TITLE
fire lockers no longer have face hiding gas masks

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes the gas mask in fire lockers the unused glass gas mask that only hides eyes rather then full face

## Why It's Good For The Game

Makes the more hiding gas masks maints loot/cargo/legit any other place you get a gas mask and not a free thing in the halls for anyone to have. - Keep in mind you can spawn with bananda's that do the same to hide face

## Changelog
:cl:
balance: fire lockers now use glass based gas masks that dont hide face
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
if you can already spawn with face covering masks why wouldn't fire closets be able to have them

**Comment by necromanceranne** [comment]
I can't believe you're shooting yourself in the foot like this Trilby.

Also personally I don't see the big issue with gas masks. On box they're actually not that terribly common. It's stations like meta and delta that make them significantly more common place.

**Comment by GrayRachnid** [comment]
😂

**Review by Ghommie — CHANGES_REQUESTED** [review]
I did check `masks.dmi` and those glass gas masks are missing their own icon_state and are using the same item_state of the normal gas masks.

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -15,7 +15,8 @@
 /obj/item/clothing/mask/gas/glass
 	name = "glass gas mask"
 	desc = "A face-covering mask that can be connected to an air supply. This one doesn't obscure your face however." //More accurate
-	icon_state = "gas_clear"
+	icon_state = "gas_mask"
+	icon_state = "gas_mask"
```
Comment (Ghommie) [inline code/modules/clothing/masks/gasmask.dm:19]: same values as parent, unnecessary.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Well, now these glass gas masks has the exact same visual aspect of normal ones. It's going to be fairly unintuitive and confusing.

**Comment by silicons** [comment]
unnecessary nerf/change to face hiding, I don't see how it's a problem.
Closing the PR in a day or two unless someone else thinks this is a good idea.

**Comment by Ghommie** [comment]
Nah, right now this is not a good idea.

File: code/modules/clothing/masks/gasmask.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/clothing/masks/gasmask.dm:19]: This is using the old one, not alt ya goof

**Pull request closed without merging**